src/integrations/meilisearch.ts

```diff
@@ -0,0 +1,32 @@
+import { retryFetchParse } from './retry';
+import { fetchOptions } from '../http';
+import { Headers } from 'node-fetch';
+
+const meiliOrigin = process.env.MEILI_ORIGIN;
+const meiliToken = process.env.MEILI_TOKEN;
+const meiliIndex = process.env.MEILI_INDEX;
+
+type Meili = {
+  post_id: string;
+  title?: string;
+};
+interface MeiliResponse {
+  hits: Meili[];
+}
+export const searchMeili = async (params: string): Promise<Meili[]> => {
+  const headers = new Headers({
+    Authorization: `Bearer ${meiliToken}`,
+  });
+  const requestOptions = {
+    method: 'GET',
+    headers,
+  };
```

Comment:
I would advise using `POST` endpoint in the future, it has a lot more options and capabilities. https://www.meilisearch.com/docs/reference/api/search#search-in-an-index-with-post

Reply:
Nice both seem to have many options already 🤯
Lets see how poc goes if good we can switch 👍